Develop an SQL schema for SOPs

```diff
@@ -0,0 +1,4351 @@
+--
+
+ALTER TABLE ONLY "PQP"."PROTEIN"
+    ADD CONSTRAINT "PROTEIN_pkey" PRIMARY KEY ("PROTEIN_ID");
+
+
+--
+-- TOC entry 2819 (class 2606 OID 17377)
+-- Name: TRANSITION TRANSITION_pkey; Type: CONSTRAINT; Schema: PQP; Owner: postgres
+--
+
+ALTER TABLE ONLY "PQP"."TRANSITION"
+    ADD CONSTRAINT "TRANSITION_pkey" PRIMARY KEY ("TRANSITION_ID");
+
+
+--
+-- TOC entry 2821 (class 2606 OID 18202)
+-- Name: FeatureXML FeatureXML_pkey; Type: CONSTRAINT; Schema: XML; Owner: postgres
+--
+
+ALTER TABLE ONLY "XML"."FeatureXML"
+    ADD CONSTRAINT "FeatureXML_pkey" PRIMARY KEY ("mzTab-ID");
+
+
+--
+-- TOC entry 2807 (class 2606 OID 17706)
+-- Name: TraML TraML_pkey; Type: CONSTRAINT; Schema: XML; Owner: postgres
+--
+
+ALTER TABLE ONLY "XML"."TraML"
+    ADD CONSTRAINT "TraML_pkey" PRIMARY KEY ("mzTab-ID");
+
+
+--
+-- TOC entry 2831 (class 2606 OID 17311)
+-- Name: PSM Section PSM Section_accession_fkey; Type: FK CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."PSM Section"
+    ADD CONSTRAINT "PSM Section_accession_fkey" FOREIGN KEY (accession) REFERENCES "MzTab"."Peptide Section"(accession) NOT VALID;
+
+
+--
+-- TOC entry 2830 (class 2606 OID 17296)
+-- Name: Peptide Section Peptide Section_accession_fkey; Type: FK CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Peptide Section"
+    ADD CONSTRAINT "Peptide Section_accession_fkey" FOREIGN KEY (accession) REFERENCES "MzTab"."Protein section"(accession) NOT VALID;
+
+
+--
+-- TOC entry 2829 (class 2606 OID 17281)
+-- Name: Protein section Protein section_mzTab-ID_REFS_fkey; Type: FK CONSTRAINT; Schema: MzTab; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab"."Protein section"
+    ADD CONSTRAINT "Protein section_mzTab-ID_REFS_fkey" FOREIGN KEY ("mzTab-ID_REFS") REFERENCES "MzTab"."Metadata_Proteomics"("mzTab-ID") NOT VALID;
+
+
+--
+-- TOC entry 2832 (class 2606 OID 18255)
+-- Name: Small Molecule Evidence (SME) Section Small Molecule Evidence (SME) Section_SME_ID_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Evidence (SME) Section"
+    ADD CONSTRAINT "Small Molecule Evidence (SME) Section_SME_ID_fkey" FOREIGN KEY ("SME_ID") REFERENCES "MzTab-M"."Small Molecule Feature (SMF) Section"("SME_ID_REFS") NOT VALID;
+
+
+--
+-- TOC entry 2833 (class 2606 OID 18248)
+-- Name: Small Molecule Feature (SMF) Section Small Molecule Feature (SMF) Section_SMF_ID_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Feature (SMF) Section"
+    ADD CONSTRAINT "Small Molecule Feature (SMF) Section_SMF_ID_fkey" FOREIGN KEY ("SMF_ID") REFERENCES "MzTab-M"."Small Molecule Section"("SMF_ID_REFS") NOT VALID;
+
+
+--
+-- TOC entry 2828 (class 2606 OID 18261)
+-- Name: Small Molecule Section Small Molecule Section_SMF_ID_REFS_fkey; Type: FK CONSTRAINT; Schema: MzTab-M; Owner: postgres
+--
+
+ALTER TABLE ONLY "MzTab-M"."Small Molecule Section"
```

Comment:
Can you look into some of the automated UML diagram tools for postgres?  The database schema is getting quite complicated and it will make things a big easier to review in particular to the foreign keys if we are able to visualize the relationships.